nni/algorithms/compression/v2/pytorch/pruning/basic_pruner.py

```diff
@@ -379,6 +379,8 @@ def trainer(model: Module, optimizer: Optimizer, criterion: Callable[[Tensor, Te
         The criterion function used in trainer. Take model output and target value as input, and return the loss.
     training_epochs : int
         The epoch number for training model to sparsify the BN weight.
+    scale : float
+        Penalty parameters for sparsification.
```

Comment:
better to explain more on this parameter. it is hard to understand with just four words

Reply:
I would fix this ASAP